core/src/main/java/org/apache/carbondata/core/util/ByteUtil.java

```diff
@@ -733,4 +733,12 @@ public static double toXorDouble(byte[] value, int offset, int length) {
   public static float toXorFloat(byte[] value, int offset, int length) {
     return Float.intBitsToFloat(toXorInt(value, offset, length));
   }
+
+  public static int[] toIntArray(byte[] data, int size) {
```

Comment:
need to optimize the name of this method since it handles for 3-bytes-int

Reply:
updated method name